client/state/ui/README.md

```diff
@@ -2,39 +2,3 @@ UI State
 ========
 
 A module for managing user interface state.
-
-## Actions
```

Comment:
Yeah, I like the idea of docs here, though in my experience it's more likely that these won't be well-maintained which could cause even more confusion (as evidenced by my lack of documenting #2080). The extended description of 1c981e846316039a8d188fc59b4e57ebec30e5b0 highlights this concern. It would be nice if we had some automated documentation, since all of the important information is already captured by the JSDoc of the functions themselves.

Maybe I'm being a bit defeatist :smile: